Added Static and SemVer tests docs (#7951)
magento/devdocs: src/guides/v2.3/test/svc/semver_test_execution.md

---
group: testing
title: Running Semantic Version Checker
functional_areas:
  - Testing
  - test
---

Executing the Magento 2 Semantic Version Checker is a simple process, but it requires an external tool.

### Running Semantic Version Checker on local changes

To run the Semantic Version Checker on local changes:

1. Clone the `magento-semver` tool to your local machine:

   ```bash
   git clone git@github.com:magento/magento-semver.git
   ```

1. Navigate to the cloned repository and run `composer install` to install the project dependencies:

   ```bash
   cd magento-semver && composer install
   ```

1. To run the Semantic Version Checker, we need one folder with the feature changes and another folder with mainline code (without changes),
so you may need to clone the Magento repository to a separate folder to perform the comparison.

1. Navigate the `magento-semver` folder and run the Semantic Version Checker compare command:

   ```bash
   bin/svc compare ../magento2-mainline ../magento2
   ```

The first parameter is the mainline code without any changes, and the second parameter is the path to the folder with your changes.

> This command you output the results of the Semantic Version Checker tool.

Reviewer: Where do the results output to? Is it obvious?

Author: The output is the result of the CLI command in this case, I guess there is a typo there, it should be:
```This command will output the results of the Semantic Version Checker tool.```

Maybe we can make it more explicit by saying:
```This command will output on the console the results of the Semantic Version Checker tool.```

What do you think @dobooth ?

Reviewer: The results of the Semantic Version Checker are outputted to the console.

> This command you output the results of the Semantic Version Checker tool.

Author: ```suggestion
The results of the Semantic Version Checker are outputted to the console.
```

---
group: testing
title: Running Semantic Version Checker
functional_areas:
  - Testing
  - test
---

Executing the Magento 2 Semantic Version Checker is a simple process but, it requires an external tool to be done.

### Running Semantic Version Checker on local changes

To run the Semantic Version Checker on local changes, you need to:

**Step 1.** Clone the `magento-semver` tool to your local machine:

```bash
git clone git@github.com:magento/magento-semver.git
```

**Step 2.** Navigate to the cloned repository and run composer install to install the project dependencies:

```bash
cd magento-semver && composer install
```

**Step 3.** In order to run the Semantic Version Checker we need to have one folder with the feature changes and another folder with mainline code (without changes),
so you may need to clone the Magento repository to a separate folder to perform the comparison.

**Step 4.** Navigate the `magento-semver` folder and run the Semantic Version Checker compare command:

```bash
.bin/svc compare ../magento2-mainline ../magento2
```

> The first parameter is the mainline code (2.3-develop branch) without any changes, and the second parameter is the path to the Magento with your feature changes.

Reviewer 2: Branches are not mentioned as this constantly changes. Please rework to not mention a branch name.
